src/Components/Web.JS/src/Services/ComponentDescriptorDiscovery.ts

```diff
@@ -26,14 +26,14 @@ function discoverServerComponents(document: Document): ServerComponentDescriptor
   return discoveredComponents.sort((a, b): number => a.sequence - b.sequence);
 }
 
-const blazorStateCommentRegularExpression = /^\s*Blazor-Component-State:(?<state>[a-zA-Z0-9\+\/=]+)$/;
+const blazorStateCommentRegularExpression = /^\s*Blazor-Component-State:(?<state>[a-zA-Z0-9+/=]+)$/;
 
```

Comment:
https://eslint.org/docs/rules/no-useless-escape#disallow-unnecessary-escape-usage-no-useless-escape

Reply:
Doesn't this un-escape `/`? Is that a thing?

Reply:
Because we start with `/^` and end with `$/` I think it follows different rules. Es-lint is insisting functionality is maintained

![Maintains](https://user-images.githubusercontent.com/14852843/135695679-62c68ac8-cebc-4d06-a838-f05389adc7cd.gif)